### ISSUE TITLE
Version 0.5.4.0 fails to compile with GHC 7.4.2

### ISSUE THREAD

**Issue by peti** [haskell/primitive#16]
A complete build log is at http://hydra.cryp.to/build/201968/nixlog/1/raw

**Comment by peti** [comment]
Any news on this?

**Comment by hvr** [comment]
Fixing this is probably as easy as

```diff
diff --git a/test/main.hs b/test/main.hs
index 2e7ef0b..3289b43 100644
--- a/test/main.hs
+++ b/test/main.hs
@@ -15,7 +15,7 @@ main = do
             case writeArray arr 0 'B' of
                 IO f ->
                     case f realWorld# of
-                        _ -> ()
+                        (# _, _ #) -> ()
     c1 <- readArray arr 0
     return $! unit
     c2 <- readArray arr 0
```

**Comment by peti** [comment]
OK. Is there anything I can do to help getting a new release to Hackage that has this fix included?

**Comment by RyanGlScott** [comment]
This was fixed in #19. The current version on Hackage ([0.6.1.0](http://hackage.haskell.org/package/primitive-0.6.1.0)) also contains this fix, so I'm closing.
